src/main/java/com/team1/dealerApp/subscription/Subscription.java

```diff
@@ -0,0 +1,36 @@
+package com.team1.dealerApp.subscription;
+
+import com.team1.dealerApp.user.User;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Subscription {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+
+    @Column(name = "price", nullable = false)
+    private Double price;
+
+    @Column(name = "start_date")
+    private LocalDate startDate;
+
+    @Column(name = "end_date")
+    private LocalDate endDate;
+
+    @Column(name = "status")
+    private boolean status;
+
+    @ManyToOne
```

Comment:
La relazione dovrebbe essere una many to many: uno user può avere più abbonamenti, e un abbonamento può avere più user associati.